lib/widgets/LocationGPS.dart

```diff
@@ -0,0 +1,32 @@
+import 'dart:async';
+import 'package:flutter/services.dart';
+
+class Location {
+  static const MethodChannel _channel = const MethodChannel('lyokone/location');
+  static const EventChannel _stream = const EventChannel(
+      'lyokone/locationstream');
+
+  Stream<Map<String, double>> _onLocationChanged;
+
+  Future<Map<String, double>> get getLocation =>
+      _channel.invokeMethod('getLocation');
+
+  Stream<Map<String, double>> get onLocationChanged {
+    if (_onLocationChanged == null) {
+      _onLocationChanged =
+          _stream.receiveBroadcastStream();
+    }
+    return _onLocationChanged;
+  }
+}
+  /*
+  As Reference!
```

Comment:
I would keep the comment, but I would use this [style](https://www.dartlang.org/guides/language/effective-dart/documentation#doc-comments). It is quite useful since you can't figure out that information from the API only.

Reply:
@Pacane it's on the readme, it can go.

Reply:
oh, right.